### PR TITLE
[Issue #200]Fixes regex that verifies repository format

### LINKED ISSUE #200: RepositoryFormatVerification should allow dashes in usernames

### PR DESCRIPTION
**Pull request by xjusko**
fixes #200

### CODE REVIEW

File: tyr-core/src/main/java/org/jboss/tyr/model/Utils.java [hunk 1]
```diff
@@ -50,6 +50,10 @@
     public static final String MESSAGE = "message";
     public static final String URL = "url";
 
+    //Repository verification regex
+    public static final String REPOSITORY_FORMAT_VERIFICATION_REGEX =
+            "^([a-zA-Z0-9](?:-?[a-zA-Z0-9]){0,38})/[a-zA-Z0-9_.-]{1,100}$";
```
Comment (xstefank) [inline tyr-core/src/main/java/org/jboss/tyr/model/Utils.java:55]: just wondering if you found this limitations somewhere? Are these official rules?

**Review by xstefank — APPROVED** [review]
LGTM

File: tyr-core/src/main/java/org/jboss/tyr/model/Utils.java (hunk 1 above)
Comment (xjusko) [reply tyr-core/src/main/java/org/jboss/tyr/model/Utils.java:55]: Numbers of characters are official, username format also official. Googled repository name format and tested it. (you can also choose other characters in repository name but they get replaced by `-`)

Comment (xjusko) [reply tyr-core/src/main/java/org/jboss/tyr/model/Utils.java:55]: > 
> 
> just wondering if you found this limitations somewhere? Are these official rules?

Max characters are official, username format also official. Googled repository name format and tested it. (you can also choose other characters in repository name but they get replaced by `-`)

**Pull request merged**